### ISSUE TITLE
Distributed Inception Model Training - Use updated SyncReplicasOptimizerV2

### ISSUE THREAD

**Issue by agnusmaximus** [tensorflow/models#562]
Top level directory: models/inception

I was wondering if the inception distributed train script could be updated to use the SyncReplicasOptimizerV2, which was recently pushed to tensorflow. This avoids some edge cases of stale gradients being applied to the model. 

I think the main changes would be 
1) Remove all references to clean_up_op
2) Remove 'replica_id=FLAGS.task_id,' argument from the call to create SyncReplicasOptimizerV2
3) Set local_init_op and ready_for_local_init_op as shown in the example: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/training/sync_replicas_optimizer.py#L126. These are then passed to the Supervisor.
4) Somehow get the V2 to use 'dtype=tf.int64,' for self._local_step

If I'm missing anything required to successfully switch over to using the V2 SyncReplicasOptimizer, definitely let me know since I'm also playing with this.

**Comment by asimshankar** [comment]
Hi @agnusmaximus . We probably do not want to change the code in the models/inception directory right now since we want to keep the model there close to what we did as described in the paper.

If you would like to try out inception with SyncReplicaOptimizerV2 yourself, that's great and we'd love to hear about your results. However, questions on how to use it are better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) as we'd like to keep the github issues focused on bugs and feature requests.

Thanks much!
